lib/TelegramHandlers/TgHelpers.js

```diff
@@ -25,5 +26,8 @@ module.exports.GetFullUserName = function(from) {
         return from.first_name;
     }
 
-    return from.first_name + " (@" + from.username + ")";
+    // Add ZWP so users with same username across bridge don't ping themselves
```

Comment:
Thanks for the useful comment, this is the kind of thing we would forget why we did it in two years.